projects/rhcos/roles/roles.yaml

```diff
@@ -0,0 +1,55 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ProjectRequest
+  apiVersion: v1
+  metadata:
+    name: rhcos
+  displayName: Red Hat CoreOS
+  description: Build/test for Red Hat CoreOS
+
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    name: team-redhat-coreos-admins
+    namespace: rhcos
+  roleRef:
+    name: admin
+  userNames:
+    - ashcrow
+    - cgwalters
+    - dustymabe
+    - nstielau
+
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    name: team-redhat-coreos
+    namespace: rhcos
+  roleRef:
+    name: edit
+  userNames:
+    - ashcrow
+    - dm0-
+    - sdemos
+    - cgwalters
+    - yuqi-zhang
+    - miabbott
+
```

Comment:
Want to add @kikisdeliveryservice while we are at it?

Reply:
Done, thanks!

Reply:
thanks @miabbott & @cgwalters !